### PR TITLE
Fix twofactor app translations

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: translations/config.json [hunk 1]
```diff
@@ -88,8 +88,11 @@
         "nextcloud tasks",
         "nextcloud terms_of_service",
         "nextcloud text",
+        "nextcloud twofactor_gateway",
+        "nextcloud twofactor_nextcloud_notification",
         "nextcloud twofactor_totp",
         "nextcloud twofactor_u2f",
+        "nextcloud twofactor_webauthn",
```
Comment (ChristophWurst) [inline translations/config.json:95]: let's wait for https://github.com/nextcloud/twofactor_webauthn/pull/74

Comment (nickvergessen) [reply translations/config.json:95]: and then also remove twofactor_u2f

Comment (ChristophWurst) [reply translations/config.json:95]: the app won't die yet. we still also need to ship an update with a warning and the documented migration path

File: translations/config.json [hunk 2]
```diff
@@ -117,7 +119,6 @@
         "LibreSign libresign",
         "mwalbeck nextcloud-breeze-dark",
         "nickv-nextcloud event_update_notification",
```
Comment (rakekniven) [inline translations/config.json:121]: Hello @nickvergessen , looks like you transferred repo "event_update_notification" to "nextcloud/" as well, or?

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: translations/config.json (hunk 2 above)
Comment (rakekniven) [reply translations/config.json:121]: Maybe this should be changed in another commit.

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**